src/main/java/org/pac4j/jax/rs/pac4j/JaxRsApplicationLogoutLogic.java

```diff
@@ -0,0 +1,16 @@
+package org.pac4j.jax.rs.pac4j;
+
+import org.pac4j.core.engine.DefaultApplicationLogoutLogic;
+
+/**
+ * @author Victor Noel - Linagora
+ * @since 1.1.1
+ * 
+ */
+public class JaxRsApplicationLogoutLogic<C extends JaxRsContext> extends DefaultApplicationLogoutLogic<Object, C> {
```

Comment:
Yes, it's always a matter of defining a profile manager factory (not only a `ProfileManager`).

@millross OK, I see your point. So we would keep the ability to set the profile manager factory at the logics level (for the pac4j implementation developers) and add a profile manager factory in the `Config` as well (for the pac4j users) which has the priority over the previous one. It sounds good. @victornoel are you in line? So you could remove the specific JAX-RS logics.

It shouldn't be a breaking change and not too complicated. I'll try to do that in 1.9.6 as well. I let you know.

Tracked here: https://github.com/pac4j/pac4j/issues/818